docker-compose.yml

```diff
@@ -1,7 +1,8 @@
-version: '2.1'
+version: '3.0'
 
 services:
   localstack:
+    container_name: "${LOCALSTACK_DOCKER_NAME-localstack}"
```

Comment:
Can we use the default name `localstack_main`? This is the container name we also use when starting LocalStack via the CLI.